### DISCUSSION TITLE
Playground questions

### DISCUSSION THREAD

**Discussion by mikygit** [open-webui/open-webui#3076] [Q&A]
Hello,
 I'm not sure I fully understand how playground is working. It looks like it's bundled to a model in a fixed manner i.e: on a per model basis

My use case would be to be able to set the selected model parameters (temperature, ....) live, within the playground itself, on a per message basis. 

Is this already possible?
Or is this considered at all in the dev roadmap?

Thanx in advance for your answers ;-)

**Comment by reminx** [comment]
I also wanna know that!

**Comment by jtc1246** [comment]
Can use this, https://github.com/jtc1246/openai-playground.
